src/multidoc-manager.js

```diff
@@ -104,7 +110,9 @@ export class MultidocManager {
       /* opt_isRuntimeCss */ true
     );
     // Instal doc services.
-    AMP.installAmpdocServices(ampdoc);
+    // AMP.installAmpdocServices(ampdoc);
+    // installAmpdocServicesForEmbed(ampdoc); // should be this?
+    installAmpdocServicesForShadowEmbed(ampdoc);
```

Comment:
I believe we need to call the same `AMP.installAmpdocServices` if we're in a non-embeded shadow, right?